### ISSUE TITLE
"Open links in new tab" Not Working

### ISSUE THREAD

**Issue by kwehner** [ujain1999/minimal_newtab#82]
I have "Open links in new tab" turned on in Settings > Keyboard navigation, but it doesn't work. Links always open in the same tab. I'm using extension version 1.5.1 in Microsoft Edge version 149.0.4022.62 for macOS.

**Comment by SQNNET** [comment]
Checked in on this, there might be a (justified) misunderstanding on how this setting works. Clicking a bookmark seems to always pull it up in the same tab; this setting specifically changes behavior for keyboard-only navigation. Just to be sure, try:

1) Toggling `Enable Keyboard Navigation` and making sure a keyboard shortcut is set
2) Opening the command overlay by inputting the set shortcut
3) Using keyboard navigation (arrow key down, search, etc) to open a bookmark

I was able to get expected behavior this way, and I believe it's intentional that this feature is available for keyboard navigation only, but the way it's implemented and shown is definitely confusing. Even though it might not technically be a bug, it's probably worth repackaging this feature as a "General" setting and just applying it to both clicks and keyboard navigation IMO.

**Comment by ujain1999** [comment]
That's a good shout, will fix in the next update.
